modules/vantagecloud-lake/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,149 @@
+[source, bash]
+----
+git clone https://github.com/Teradata/lake-demos.git
+cd lake-demos
+----
+== Mount Docker files 
+To launch Teradata VantageCloud Lake demos, we need the https://hub.docker.com/r/teradata/jupyterlab-extensions[Teradata Jupyter Extensions for Docker]. These extensions provide the SQL ipython kernel, utilities to manage connections to Teradata, and the database object explorer to make you productive while interacting with the Teradata database.   
+----
+docker pull teradata/jupyterlab-extensions
+----
+Next, start a container and bind it to the existing lake-demos directory. Choose the appropriate command based on your operating system: 
+[tabs]
+====
+Windows::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $(PWD):/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+macOS::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $PWD:/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+Linux::
++
+[source,bash]
+----
+docker run -e "accept_license=Y" -p 127.0.0.1:8888:8888 -v $PWD:/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+====
+Take note of the resulting URL and token; you’ll need them to establish the connection from Visual Studio Code.
+
+image::{dir}/terminal.png[terminal.png,align="center", width=80%]
+
+== Visual Studio Code Configuration
+Open `lake-demos` project directory in Visual Studio Code. The repository contains the following project tree: 
+
+LAKE_DEMOS
+
+* https://github.com/Teradata/lake-demos/tree/main/UseCases[UseCases]
+** https://github.com/Teradata/lake-demos/blob/main/0_Demo_Environment_Setup.ipynb[0_Demo_Environment_Setup.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/1_Load_Base_Demo_Data.ipynb[1_Load_Base_Demo_Data.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Engineering_Exploration.ipynb[Data_Engineering_Exploration.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Science_OAF.ipynb[Data_Science_OAF.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Demo_Admin.ipynb[Demo_Admin.ipynb]
+* https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]
+
+=== Edit vars.json file 
+Edit the *https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]* file to include the required credentials to run the demos +
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+| *"host"* 
+| Public IP value from your VantageCloud Lake environment
+| *"UES_URI"* 
+| Open Analytics from your VantageCloud Lake environment
+| *"dbc"*
+| The master password of your VantageCloud Lake environment.
+|====
+* You'll see that in the sample vars.json, the passwords of all users are defaulted to "password", this is just for matters of the sample file, you should change all of these password fields to strong passwords, secure them as necessary and follow other password management best practices.
```

Comment:
I don't think this is a bullet. If you want to make it a bullet, then IMO all other steps should become bullets as well. Perhaps a regular paragraph plus a warning/attention callout will be a better option.